src/features/transferHotspot/TransferHotspot.tsx

```diff
@@ -49,6 +53,11 @@ const TransferHotspot = () => {
     const pendingTxn = await submitTxn(signedTxnString)
     setHash(pendingTxn.hash)
     setLoading(false)
+
+    // Segment track for Hotspot transfer
+    track(HotspotEvents.TRANSFER_SUBMITTED, {
```

Comment:
In a case where they cancel the transfer request on Helium app, how do you track that.

Also shouldn't there be some sort of link between the transfer request and submission?

Reply:
In case the transfer request cancelled, simply there will be no 'TRANSFER_SUMITTED` event tracked.